### PR TITLE
Added white background to any user or organization avatar

### LINKED ISSUE #5500: Set white background to header avatar
In order to avoid blank spaces in PNG or GIF images, we should add a white background to the container of the avatar shown on the header:

screen shot 2015-09-14 at 15 41 59

### PR DESCRIPTION
**Pull request by xavijam**
screen shot 2015-09-14 at 19 45 50
screen shot 2015-09-14 at 19 34 31
screen shot 2015-09-14 at 19 34 35

Fixes #5500.

Hello Hound... 
REVIEWER: @MariaCheca

### CODE REVIEW

**Comment by MariaCheca** [comment]
Could you deploy it to staging to try it out by myself? Also @houndci is right :)

**Comment by xavijam** [comment]
If you check the changes, I only moved properties, not added anything, !important was there before I arrived ;).

File: app/assets/stylesheets/common/account_forms.css.scss [hunk 1]
```diff
@@ -471,9 +582,9 @@ $sLabel-width: 140px;
   border: 1px solid #F2C000;
   border-radius: 20px;
   background: #F6D35C;
-  line-height: $sLineHeight-normal!important;
```
Comment (xavijam) [inline app/assets/stylesheets/common/account_forms.css.scss]: cc @MariaCheca

**Comment by xavijam** [comment]
Deploying it in rui servers.

**Comment by MariaCheca** [comment]
Tested and approved :+1:

**Pull request merged**